etc/boot

```diff
@@ -157,14 +160,22 @@ if args.config:
     if VERB: print INFO, "Using config file", config
 
 hyper_name = "qemu"
-if args.solo5:
-    hyper_name = "solo5"
+if args.solo5_hvt:
+    hyper_name = "solo5-hvt"
     os.environ['PLATFORM'] = "x86_solo5"
     solo5_hvt = INCLUDEOS_PREFIX + "/x86_64/bin/solo5-hvt"
 
     subprocess.call(['chmod', '+x', solo5_hvt])
     subprocess.call(['sudo', INCLUDEOS_PREFIX + "/scripts/solo5-ifup.sh" ])
 
+elif args.solo5_spt:
+    hyper_name = "solo5-spt"
+    os.environ['PLATFORM'] = "x86_solo5"
+    solo5_spt = INCLUDEOS_PREFIX + "/x86_64/bin/solo5-spt"
+
+    subprocess.call(['chmod', '+x', solo5_spt])
+    subprocess.call(['sudo', INCLUDEOS_PREFIX + "/includeos/scripts/solo5-ifup.sh" ])
```

Comment:
This should be "INCLUDEOS_PREFIX + "/scripts/solo5-ifup.sh" like above. The error comes up when running in rpr

Reply:
Thanks, I've fixed that now